src/components/UI/Charts/Sunburst/Sunburst.tsx

```diff
@@ -22,10 +34,26 @@ type SunburstProps = {
 
 const pickPhenotypeInfo = (p: Phenotype) => ({
   title: p.title,
+  key: p.key,
   results: p.results,
   exactTagCount: p.exactTagCount,
 });
 
+export const hpoTreeTitleFormat = (splitPheno: PhenotypeSplit, currentTerm: string) => {
+  const currentPhenoCode = splitHPOTerm(currentTerm).code;
+  return (
+    <div
+      className={'hpo-tree-name'}
+      style={{
+        fontWeight: currentPhenoCode !== splitPheno.code ? 400 : 'bold',
```

Comment:
I would use css with a condition vs style and put my condition in a var => 
const isCurrentPhenoCode = currentPhenoCode !== splitPheno.code;

className={'hpo-tree-name ${isCurrentPhenoCode ? 'myClassName': ''}'}